integration/simulation/transaction_injector.go

```diff
@@ -166,11 +167,32 @@ func (ti *TransactionInjector) issueRandomTransfers() {
 		for len(ti.wallets.SimObsWallets) > 1 && fromWallet.Address().Hex() == toWallet.Address().Hex() {
 			toWallet = ti.rndObsWallet()
 		}
-		tx := ti.newObscuroTransferTx(fromWallet, toWallet.Address(), testcommon.RndBtw(1, 500))
+		tx := ti.newObscuroTransferTx(fromWallet, toWallet.Address(), testcommon.RndBtw(1, 3))
 		signedTx, err := fromWallet.SignTransaction(tx)
 		if err != nil {
 			panic(err)
 		}
+
+		callMsg := &ethereum.CallMsg{
+			From:       fromWallet.Address(),
+			To:         signedTx.To(),
+			Gas:        signedTx.Gas(),
+			GasPrice:   signedTx.GasPrice(),
+			GasFeeCap:  signedTx.GasFeeCap(),
+			GasTipCap:  signedTx.GasTipCap(),
+			Value:      signedTx.Value(),
+			Data:       signedTx.Data(),
+			AccessList: signedTx.AccessList(),
+		}
+
+		gasEstimate, err := ti.rpcHandles.ObscuroWalletRndClient(fromWallet).EstimateGas(ti.ctx, callMsg)
+		if err != nil {
+			// time.Sleep(10 * time.Minute)
```

Comment:
Did you mean to leave this in?

Reply:
Oops, thank you 🙏

Reply:
Update: I've disabled this as the nonce logic is impacting the ability to do a proper gas estimate and THEN use that estimate in a Tx. 
Will create a new PR with it.

The newObscuroTransferTx method should not auto-fetch and increment the nonce because if a new tx is created before the newObscuroTransferTx is issued, then the nonces collide.


In this particular case, this flow happens:
```
-tx1  = newObscuroTransferTx     
-gas1 = EstimateGas.     |  at the same time tx2 = newObscuroWithdrawalTx 
-tx1' = tx1 with gas1    |  tx2 panic: transaction injector could not retrieve nonce after thirty seconds for address. Local nonce was 1, remote nonce was 0)
-Issue tx1'   (never happens)
```

Reply:
Adding a cleaner example
```
If `newObscuroTransferTx` creates Tx1 then local nonce = 1  and it issues it then remote nonce =1 
If `newObscuroWithdrawalTx` creates Tx2 then local nonce =2 and it issues it then remote nonce = 2

BUT
If `newObscuroTransferTx` creates Tx1 then local nonce = 1  and it does not issue in time then remote nonce =0
If `newObscuroWithdrawalTx` creates Tx2 then local nonce =2 but when it issues it then remote nonce = 1
```